OpenRA.Mods.Common/Traits/AutoTarget.cs

```diff
@@ -130,11 +133,23 @@ public void TickIdle(Actor self)
 			if (Stance < UnitStance.Defend || !Info.TargetWhenIdle)
 				return;
 
-			var allowMove = Info.AllowMovement && Stance != UnitStance.Defend;
-			if (attackFollows.All(a => a.IsTraitDisabled || !a.IsReachableTarget(a.Target, allowMove)))
+			bool allowMove;
+			if (ShouldAttack(out allowMove))
 				ScanAndAttack(self, allowMove);
 		}
 
+		bool ShouldAttack(out bool allowMove)
+		{
+			allowMove = Info.AllowMovement && Stance != UnitStance.Defend;
```

Comment:
Is it really worth to have this as `out` parameter just to save us writing this line two times? As as far as I see `allowMove` isn't changed anymore in this funtion.